rational/tests/convert.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::approx_constant)]
```

Comment:
Same, should use `core::f32::consts::PI` in this file. But actually the line triggering this lint is a debug line I accidentally put into the test. Could you please help turning that into an actual test?